multinode-demo/leader.sh

```diff
@@ -6,5 +6,13 @@ export RUST_LOG=${RUST_LOG:-solana=info}
 set -x
 [[ $(uname) = Linux ]] && sudo sysctl -w net.core.rmem_max=26214400
 
+IPADDR="$(ifconfig  | grep 'inet addr:'| grep -v '127.0.0.1' | cut -d: -f2 | awk '{ print $1}')"
```

Comment:
what does ifconfig show on a Mac?  Looks like you're working really hard here...
on stock Ubuntu this doesn't work (no ifconfig)

Reply:
can you try this on your mac:
```
ifconfig | awk '/inet(6)? (addr:)?/ {print $2}'
```

Reply:
ifconfig | grep inet shows the following on Mac
inet 192.168.130.37 netmask 0xfffffc00 broadcast 192.168.131.255

On ubuntu it's this
inet addr:10.138.0.3  Bcast:10.138.0.3  Mask:255.255.255.255

Reply:
I recommend using all the IPs as match possibilities in the leader-${IP} selection

Looks like:
```IPS=$(ifconfig | awk '/inet(6)? (addr:)?/ {print $2}')
for ip in $IPS
do
  [[ -f leader-${ip}.json ]] && break;
done
echo leader-${ip}.json
```

Reply:
Sorry, I didn't understand.. leader.sh is generating the file name based on the ip addr of the machine where it's running. How is $IPS computed?

Reply:
This works on both Mac and ubuntu:
ifconfig | awk '/inet (addr)?/ {print $2}' | cut -d: -f2 | grep -v '127.0.0.1'